### PR TITLE
Updating QA bands based

### PR DESCRIPTION
**Pull request by CodeRads**
Added behaviors to jr.qa role and some additional changes to other bands

### CODE REVIEW

File: qa_bands.md [hunk 1]
```diff
@@ -20,27 +20,34 @@
...
 ### Project Scope
 * Delivers assignments with high quality, readability, and timely delivery
-* Designs solutions to medium complexity testing problems, within constraints appropriate for the platform
 * Coordinates releases to QA and stage environments
-* Coordinates execution plan for distributed QA partners
+* Coordinates and works with offsite QA partners
+* Owns components on their projects or a standalone project
 * Owns small component/area on a few projects or a standalone project
-* You build analytics requirements, dashboards, and alerts to ensure we can observe how our code performs in non-prod and prod environments
+* Start looking at dashboards,alerts to ensure code performs in non-prod and prod environments
```
Comment (sethsakamoto) [inline qa_bands.md]: I like this, but would remove the passive language of "start looking at" and replace with something like "uses".

I also would like to preserve the part about building analytics requirements, dashboards, & alerts.

File: qa_bands.md [hunk 2]
```diff
@@ -20,27 +20,34 @@
...
+* Maintain release notes 
 
 ### Behaviors
 * You're championing quality throughout your team.
 * Testing is a natural part of your workflow; you feel nervous when we're making changes.
-* You are contributing to best practices/patterns at the software level (testing practices, risk identification and management, coding practices, style guidelines, core libraries, etc.) in the interest of producing quality.
+* You are contributing to best practices/patterns at a quality level (testing practices, risk identification and management etc.) in the interest of producing quality.
```
Comment (sethsakamoto) [inline qa_bands.md]: The point of this is to collaborate with programmers on how code is written, so maybe wouldn't change it.  (original wording kind of sucks though)

File: qa_bands.md [hunk 3]
```diff
@@ -50,54 +57,51 @@
...
 ### Project Scope
 * Own testing effort and process for a few large components or projects
-* Designs solutions to high complexity problems, with high-availability and high-reliability in mind
+* Design test automation for  high complexity problems, with high-availability and high-reliability in mind
 * Instigates test design discussions, evaluates approaches, articulates goals and criteria
-* Responsible for overall quality of code, product
+* Responsible for overall quality of code for automation and the application
 * Determines appropriate tools & techniques for monitoring quality of code in production and diagnosing issues
 * Primary driver of investigating and coordinating resolution for production issues
 * Ensures proper diagnosis and approach for resolving tricky bugs
```
Comment (sethsakamoto) [inline qa_bands.md:97]: Perhaps would add one about co-driving investigation and resolution of nasty and/or high-priority bugs?

File: qa_bands.md [hunk 4]
```diff
@@ -106,9 +110,8 @@
 ## Principal QA Engineer
 ### Technical Abilities
 * Mastery of TuneIn's automation frameworks
-* Mastery of TuneIn's application and systems architecture
-* Is an authority in multiple test technical disciplines, languages, frameworks, and systems
-* Demonstrates a unique and extraordinarily effective problem solving perspective and approach
```
Comment (sethsakamoto) [inline qa_bands.md:111]: I might keep this line, or something about triaging and troubleshooting

Comment (CodeRads) [reply qa_bands.md:111]: True, but wouldn't that be something that falls under project scope as opposed to technical abilities?

File: qa_bands.md (hunk 2 above)
Comment (CodeRads) [reply qa_bands.md]: Is this better: You are contributing to best practices/patterns at a software level (testing practices, risk identification and management etc.) by collaborating with developers in the interest of producing quality.

**Pull request merged**